db/migrate/20160111111547_change_advocate_persona_type_to_external_user.rb

```diff
@@ -1,9 +1,5 @@
 class ChangeAdvocatePersonaTypeToExternalUser < ActiveRecord::Migration
   def change
-    User.all.each do |user|
-      if user.persona_type == 'Advocate'
-        user.update_column(:persona_type, 'ExternalUser')
-      end
-    end
+    execute(%q{UPDATE users SET persona_type = 'ExternalUser' WHERE persona_type = 'Advocate'} )
```

Comment:
Well I could do, but I only changed the old version so that the migrations would work now that I've added a new field to the users table - it runs all the migrations from scratch as part of the CI suite, which is probably wrong - it should be loading the database schema from schema.rb.  The old version didn't allow backwards migration and we're not going to a migration from January.